dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -76,6 +57,8 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.tags.Tag;
 
+import static org.apache.dolphinscheduler.api.enums.Status.*;
```

Comment:
Hi, please run `./mvnw spotless:apply` to auto-format your code, BTW, we do not accept wildcard import

Reply:
Thank you very much, the open source road is full of details, learned.